core/src/main.cpp

```diff
@@ -241,6 +241,9 @@ constexpr auto default_jsonschema() -> std::string_view
                                 "access_token_secret": {
                                     "type": "string"
                                 },
+                                "access_token_validation_method": {
```

Comment:
Should we to add this to the list of "required" keys? The implementation will throw an exception if it is not present.

Reply:
Done.

I see the following failure now with the change:
```
...
Configuration failed validation.
[
    {
        "valid": false, 
        "evaluationPath": "/properties/http_server/properties/authentication/properties/openid_connect/required", 
        "schemaLocation": "https://schemas.irods.org/irods-http-api/config.json#/properties/http_server/properties/authentication/properties/openid_connect/required", 
        "instanceLocation": "/http_server/authentication/openid_connect", 
        "error": "Required property 'access_token_validation_method' not found."
    }
]
```